app/src/highlighter/index.ts

```diff
@@ -149,6 +150,10 @@ const extensionModes: ReadonlyArray<IModeDefinition> = [
       '.cpp': 'text/x-c++src',
       '.hpp': 'text/x-c++src',
       '.cc': 'text/x-c++src',
+      '.hh': 'text/x-c++src',
+      '.hxx': 'text/x-c++src',
+      '.cxx': 'text/x-c++src',
+      '.cmake': 'text/x-c++src',
```

Comment:
https://codemirror.net/5/mode/cmake/index.html defines this as `text/x-cmake`

Reply:
Removed `x-cmake` from CLike and created a new install specific for CMake.